guides/CorrectingTrackLabellingSteps.md

```diff
@@ -0,0 +1,187 @@
+
+- Make sure the track id is shown instead of the bounding box number.
+
+  - To do so, press the up key. The Up key toggles between `region label cleared `or `region label set to region attribute [track]`. To show the track ID, we want the second one `region label set to region attribute [track]`.
+
+> [!TIP]
+> Pressing the bottom key will change the colour of the bounding box, but notice that the colour is not consistent across frames. However, it may be helpful to disentangle boxes when they are overlapping.
+
+#### 4.1 Recommended approach:
+
+> [!CAUTION]
+> Remember there is no undo action, so make sure you save often, especially before correcting ID switches.
+
+- Select a crab ID to follow across frames.
+
+  - You can highlight the crab with a certain ID by using the Annotation editor. To show this editor, click under `Attributes` > `Toggle Annotation Editor`. Then click on the row with the track ID of interest.
+  - Be careful now, when it is highlighted, if you press right key, the bounding box will be shifted to the right. Just click anywhere to disable the highlight.
+
+- If the crab you are following has no bounding box in one of the frames: copy the box from the previous frame.
+
+  - Use `ctrl+c` to copy the box from the same crab in the previous frame, then paste it (`ctrl+v`) on the current frame. This ensures we pass the same track ID. You can adjust the bounding box to where the crab is either by dragging with your mouse or using the arrow keys.
+
+- If the crab you are following is assigned to a new ID for a large number of frames (i.e., it has been re-IDed/there was an ID switch): change all the annotations to match the ID in the previous frames.
+
+  - In the menu bar, select the grid icon (6th icon after `Help` menu).
+    - All the frames will be shown in a grid style.
+    - You do not need to wait until the grid images are finished loading before doing any of the step below.
+  - Choose `Group by` `region [track]` - a selector for the track number will pop up next to it.
+  - In the selector, choose the track you want to edit.
+    - For example: if the original ID is 9, and the crab has now been re-IDed to 12, then choose `12/79: track = 12`.
+    - Notice that the first number shown in the dropdown menu options does not necessarily match the second one (this usually happens at the end of the list, you may see e.g. `78/79: track = 86`). The one we care about is the track ID, so the second one.
+  - Toggle the Annotation Editor to edit the tracks.
+    - You should see the Annotation Editor panel with the track ID field at the bottom of the screen.
+    - If the track ID field is not editable:
+      - Make sure the subset of frames shown in the grid are selected.
+      - If they are and the ID is still not editable, try clicking a few times in the `Toggle Annotation Editor` button (it may show something like `xxx different values:...`).
```

Comment:
```suggestion
      - If they are and the ID is still not editable, try clicking a few times on the `Toggle Annotation Editor` button (it may show something like `xxx different values:...`).
```

Reply:
Also: 🤣